dd-library-php-setup.php

```diff
@@ -69,44 +71,47 @@ function install($options)
     $interactive = empty($options[OPT_PHP_BIN]);
 
     // Preparing clean tmp folder to extract files
-    $tmpDir = sys_get_temp_dir() . '/dd-library';
-    $tmpDirTarGz = $tmpDir . '/dd-trace-php.tar.gz';
-    $tmpSourcesDir = $tmpDir . '/opt/datadog-php/dd-trace-sources';
-    $tmpExtensionsDir = $tmpDir . '/opt/datadog-php/extensions';
+    $tmpDir = sys_get_temp_dir() . '/dd-install';
+    $tmpDirTarGz = $tmpDir . "/dd-library-php-x86_64-$platform.tar.gz";
+    $tmpArchiveRoot = $tmpDir . '/dd-library-php';
+    $tmpArchiveTracerRoot = $tmpDir . '/dd-library-php/tracer';
+    $tmpBridgeDir = $tmpArchiveTracerRoot . '/bridge';
     execute_or_exit("Cannot create directory '$tmpDir'", "mkdir -p " . escapeshellarg($tmpDir));
     execute_or_exit(
         "Cannot clean '$tmpDir'",
         "rm -rf " . escapeshellarg($tmpDir) . "/* "
     );
 
     // Retrieve and extract the archive to a tmp location
-    if (isset($options[OPT_TRACER_FILE])) {
-        $tmpDirTarGz = $options[OPT_TRACER_FILE];
+    if (isset($options[OPT_FILE])) {
+        $tmpDirTarGz = $options[OPT_FILE];
     } else {
-        $url = isset($options[OPT_TRACER_URL])
-            ? $options[OPT_TRACER_URL]
-            : "https://github.com/DataDog/dd-trace-php/releases/download/" .
-            $options[OPT_TRACER_VERSION] . "/datadog-php-tracer-" .
-            $options[OPT_TRACER_VERSION] . ".x86_64.tar.gz";
+        $url = isset($options[OPT_URL])
+            ? $options[OPT_URL]
+            : "https://github.com/labbati/test-actions/releases/download/" .
```

Comment:
`test-actions` is temporary for CI, will have to be changed right before the final approval to `dd-trace-php`